libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405WING-bd/defaults.parm

```diff
@@ -0,0 +1,23 @@
+# WS2812 LED
+NTF_LED_LEN,4
+SERVO12_FUNCTION,120
+
+#Serial Port defaults
+SERIAL1_PROTOCOL 23
+SERIAL4_PROTOCOL -1
+SERIAL6_PROTOCOL 2
+
+#dshot-bidir
```

Comment:
I don't think you should be setting any of these. This could be used on a quadplane, etc, etc. Best to let the user configure them.